Projects/Features/SearchFeature/Sources/View/BeforeSearchSectionHeaderView.swift

```diff
@@ -0,0 +1,67 @@
+import DesignSystem
+import SnapKit
+import Then
+import UIKit
+
+protocol BeforeSearchSectionHeaderViewDelegate: AnyObject {
+    func tap(_ section: Int?) -> Void
+}
+
+class BeforeSearchSectionHeaderView:
```

Comment:
```suggestion
final class BeforeSearchSectionHeaderView:
```
상속을 위한 class가 아니면 final 붙이면 좋겠어요

Reply:
반영했습니다.